doc/src/manual/faq.md

```diff
@@ -645,8 +650,8 @@ generate efficient code when working with `Union{T, Nothing}` arguments or field
 To represent missing data in the statistical sense (`NA` in R or `NULL` in SQL), use the
 [`missing`](@ref) object. See the [`Missing Values`](@ref missing) section for more details.
 
-The empty tuple (`()`) is another form of nothingness. But, it should not really be thought of
-as nothing but rather a tuple of zero values.
+The empty tuple (`()`) is another form of nothingness. But, it should not really be thought
+of as nothing but rather a tuple of length zero.
```

Comment:
I don't really like either formulation particularly - I wouldn't say something like "but rather a real tuple that happens to contain no values".